mmv1/third_party/terraform/services/cloudidentity/data_source_cloud_identity_group_transitive_memberships.go.erb

```diff
@@ -0,0 +1,179 @@
+<% autogen_exception -%>
+package cloudidentity
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+<% unless version == 'ga' -%>
```

Comment:
Do we need to switch the client version? It looks like the schemas are the same between beta & GA, so we could just use the v1 client for both

Reply:
Good spot, thanks! I'll remove

Reply:
Removed in [60da2c4](https://github.com/GoogleCloudPlatform/magic-modules/pull/11337/commits/60da2c4125361dfa836c24c0818114d53f2c0132)

Reply:
Ah, this is actually needed. In the data source we use `config.NewCloudIdentityClient` and in the Beta version of the provider that function is declared using the beta version of client. I've rebased to remove the commit above